docker-storage-setup.sh

```diff
@@ -35,6 +35,39 @@ STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
 STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
+usage() {
```

Comment:
Looks like that there is code above Main script too. I probably did not pay much attention. Can we fix it.